torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -299,6 +306,22 @@ def param_to_rank(self) -> Dict[torch.Tensor, int]:
                         self._param_rank[param] = rank
         return self._param_rank
 
+    @property
+    def param_to_index(self) -> Dict[int, int]:
```

Comment:
Is there any reason that we exposed these two functions as public APIs?

Reply:
ah no, good point, just @property muscle memory I guess